Frontend/src/components/TaskList.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import CsrfToken from './CsrfToken';
+
+export interface ITask {
+  id: number;
+  title: string;
+  status: boolean;
```

Comment:
I feel like `status` could be interpreted to mean a few different things, but what we're really asking here is whether a task is completed or not, so I would say to change it to `completed` or `isCompleted`.

Reply:
Change has been made, thank you for the feedback. I wonder if I would need to change that on the `Task` model as well.

Reply:
I had to change the property on the model as well. Everything is working correctly.